docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -13,31 +13,26 @@
-   ```sh
-   cd <INSTALL_DIR>/install_dependencies/
+1. Check your operating system and its version: 
+   ```sh 
+   cat /etc/os-release
    ```
 
-2. Install the **Intel® Graphics Compute Runtime for OpenCL™** driver components required to use the GPU plugin and write custom layers for Intel® Integrated Graphics. The drivers are not included in the package. To install it, run this script:
-   ```sh
-   sudo -E ./install_NEO_OCL_driver.sh
-   ```
-   > **NOTE**: To use the **Intel® Iris® Xe MAX Graphics**, see the [Intel® Iris® Xe MAX Graphics with Linux*](https://dgpu-docs.intel.com/devices/iris-xe-max-graphics/index.html) page for driver installation instructions.
-   
-   The script compares the driver version on the system to the current version. If the driver version on the system is higher or equal to the current version, the script does 
-   not install a new driver. If the version of the driver is lower than the current version, the script uninstalls the lower version and installs the current version with your permission:
-   ![](../img/NEO_check_agreement.png) 
-
-   Higher hardware versions require a higher driver version, namely 20.35 instead of 19.41. If the script fails to uninstall the driver, uninstall it manually. During the script execution, you may see the following command line output:  
-   ```sh
-   Add OpenCL user to video group    
-   ```
-   Ignore this suggestion and continue.<br>
-   You can also find the most recent version of the driver, installation procedure and other information on the [Intel® software for general purpose GPU capabilities](https://dgpu-docs.intel.com/index.html) site.
+2. You can install the latest available **Intel® Graphics Compute Runtime for OpenCL™** for your OS, see the [Install Guides](https://dgpu-docs.intel.com/installation-guides/index.html) or choose one that we used in internal tests
+
+Operation System | Driver version
+--- |-------------------------
+Ubuntu 20.04 | [22.35.24055](https://github.com/intel/compute-runtime/releases/tag/22.28.23726)  
+Ubuntu 18.04 | [21.38.21026](https://github.com/intel/compute-runtime/releases/tag/22.28.23726)  
+CentOS 7 | [19.41.14441](https://github.com/intel/compute-runtime/releases/tag/22.28.23726)  
+RHEL 8 | [22.28.23726](https://github.com/intel/compute-runtime/releases/tag/22.28.23726)                     
 
-3. **Optional:** Install header files to allow compilation of new code. You can find the header files at [Khronos OpenCL™ API Headers](https://github.com/KhronosGroup/OpenCL-Headers.git).
+> **NOTE**: To use the **Intel® Iris® Xe MAX Graphics**, see the [Intel® Iris® Xe MAX Graphics with Linux*](https://dgpu-docs.intel.com/devices/iris-xe-max-graphics/index.html) page for driver installation instructions.
+
+Additionally, it is recommended that you refer to the [Intel® Graphics Compute Runtime Github page](https://github.com/intel/compute-runtime/) for the latest instructions and recommendations on GPU driver installation.
```

Comment:
This link (to Github) and the one in the beginning of step 2 (to dgpu-docs) are different but both are presented as something about the latest driver. Can we align this?

Reply:
> This link (to Github) and the one in the beginning of step 2 (to dgpu-docs) are different but both are presented as something about the latest driver. Can we align this?

What I meant was the Github link and the links in the table are from the same source. Now I see they are different in "tested versions" and "latest version". :)